src/api/artifact/controller.go

```diff
@@ -32,17 +36,19 @@ type Controller interface {
 	// creates it if it doesn't exist. If tags are provided, ensure they exist
 	// and are attached to the artifact. If the tags don't exist, create them first
 	Ensure(ctx context.Context, repository *models.RepoRecord, digest string, tags ...string) (err error)
-	// List artifacts according to the query and option
+	// List artifacts according to the query, specify the properties returned with option
 	List(ctx context.Context, query *q.Query, option *Option) (total int64, artifacts []*Artifact, err error)
 	// Get the artifact specified by ID
 	Get(ctx context.Context, id int64, option *Option) (artifact *Artifact, err error)
```

Comment:
could you also comment the usage of option?

Reply:
Done